Tribler/Core/torrentstore.py

```diff
@@ -130,5 +137,13 @@ def close(self):
         self._db = None
 
 
+    if LEVELDBPROVIDER == 'leveldb':
```

Comment:
Don't do this. define the function as a global one inside of the try/except block.